includes/class-sensei-course.php

```diff
@@ -3459,6 +3458,84 @@ public function log_initial_publish_event( $course ) {
 		sensei_log_event( 'course_publish', $event_properties );
 	}
 
+	/**
+	 * Setup the single course page.
+	 *
+	 * @access private
+	 */
+	public function setup_single_course_page() {
+		global $post;
+
+		// Remove legacy actions on courses with new blocks.
+		if (
+			$post
+			&& is_singular( 'course' )
+			&& ! $this->is_legacy_course( $post )
+		) {
+			$this->remove_legacy_course_actions();
+		}
+	}
+
+	/**
+	 * Adds legacy course actions.
+	 *
+	 * @param Sensei_Main $sensei Sensei object.
+	 */
+	public function add_legacy_course_hooks( $sensei ) {
+		// Legacy progress bar on the single course page.
+		add_action( 'sensei_single_course_content_inside_before', array( $this, 'the_progress_statement' ), 15 );
```

Comment:
As it's a new code, would it make sense we use the `[]` instead of `array()`?

Reply:
Switched to short-array in 7e0e6e83e57cdd2edbd146b56a633ee44a9ec4ec